tests/comfy_cli/jobs/test_jobs.py

```
@@ -344,12 +350,302 @@ def test_jobs_status_server_up_still_uses_live_snapshot(monkeypatch: pytest.Monk
+        assert result.exit_code == 0, result.output
+        data = _last_json(result.stdout)["data"]
+        assert data["status"] == "completed"
+        assert data["outputs"] == ["http://127.0.0.1:8188/view?filename=out.png"]
+        assert data["error"] is None
+        assert data["source"] == "state_file"
+        assert data["server_running"] is True
+
+    def test_non_terminal_state_file_keeps_the_code_but_enriches_details(self, monkeypatch: pytest.MonkeyPatch):
+        """Non-terminal record -> still `prompt_not_found` (callers key on the
+        code) but carrying the last-known state, like the server-down twin."""
+        from comfy_cli import jobs_state
```

Pylint (4.0.6)

[error] Unable to import 'comfy_cli'

(E0401)